spring-integration-ip/src/main/java/org/springframework/integration/ip/dsl/Tcp.java

```diff
@@ -37,7 +38,7 @@
 	 * Boolean indicating the connection factory should not use NIO
 	 * (default).
 	 */
-	public static final boolean NET = true;
+	public static final boolean NET = false;
```

Comment:
Uh! That's nasty.

I suggest to deprecate both these constants altogether because they are not used in the Framework at all.

And use just simply `true/false` in those factories.